owa/app/js/components/App.jsx

```diff
@@ -7,21 +7,22 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 import React from 'react';
-import 'bootstrap/dist/css/bootstrap.css'; 
-import Header from './common/header'
-import BodyLayout from './bodyLayout'
-import BreadCrumbs from './breadcrumbs/breadcrumbs'
+import 'bootstrap/dist/css/bootstrap.css';
+import 'sweetalert2/dist/sweetalert2.min.css';
```

Comment:
Minor: Consider following the react best practices for dependency and local imports. The styling imports should be after the local file imports. This resource explains it better https://engineering.musefind.com/our-best-practices-for-writing-react-components-dec3eb5c3fc8

Reply:
Yea you are right. I did that because it's an `absolute/package` import and it's a best practice to import them before importing the absolute packages. It's also part of the eslint rules.